frontend/src/pages/InducteeSignUpPage/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Avatar, Grid } from '@material-ui/core';
+
+import { SignUpForm } from './components/SignUpForm';
+import useStyles from './styles';
+
+import HKN_TRIDENT_LOGO from '@Images/hkn-trident.png';
+import { Card, PublicPageLayout } from '@SharedComponents';
+
+import { createNewUser } from '@Services/AuthService';
+import {
+  doSignInWithEmailAndPassword,
+  doSendVerificationEmail,
+  doSignOut,
+} from '@Services/auth';
+import { AppUserSignupRequest } from '@Services/api/models';
+
+function InducteeSignUpPage(): JSX.Element {
+  const classes = useStyles();
+
+  const handleSubmit = async (
+    values: AppUserSignupRequest,
+    setSubmitting: (_: boolean) => void
+  ) => {
```

Comment:
Looking at your code below, you really only need `email`, `password` and `graduationYear` from the form values, so you can do something like:

```
const { email, password, graduationYear } = values;
```